packages/docusaurus-theme-classic/src/theme/CodeBlock/index.tsx

```diff
@@ -244,11 +244,22 @@ export default ({
                   }
 
                   return (
-                    <div key={i} {...lineProps}>
-                      {line.map((token, key) => (
-                        <span key={key} {...getTokenProps({token, key})} />
-                      ))}
-                    </div>
+                    <pre
+                      style={{
+                        margin: '0px',
+                        padding: '0px',
+                        whiteSpace: 'pre-wrap',
+                        wordBreak: 'break-word',
+                      }}>
```

Comment:
We should use the css module styles instead of inline styles

Reply:
Oh, okay! Copy that! Next commit in a jiffy. Thank you.

Reply:
Done!!!